src/swagger/objects/document.cr

```diff
@@ -2,7 +2,8 @@ module Swagger::Objects
   struct Document
     include JSON::Serializable
 
-    property openapi = Swagger::OPENAPI_VERSION
+    @[JSON::Field(key: "openapi")]
+    property openapi_version : String = "3.0.1"
```

Comment:
- I changed the naming to make it more explicit it is a version. Otherwise, `Document#openapi : String` is not very explicit.
- it could be a `SemanticVersion`